### DISCUSSION TITLE
Refresh luge with accordion / Carousel

### DISCUSSION THREAD

**Discussion by IsaMax** [AntoineW/luge#12] [Q&A] [answered]
Hi !
I'm using Luge in a website. Actually, when i click on an accordion, i recalculate with javascript the height of the body in order to prevent overflow in the footer. But is there another solution to do that ? I have the same issue with a carousel: For instance, when a big image is lazyloaded after, how to recalculate the height of the body ? I tried the "lifecyle" but it does not work. 
Thanks ! (sorry for my english :) )

**Comment by AntoineW** [comment, marked as answer]
Hi,
Did you try emitting a resize event with the emitter like so `luge.emitter.emit('resize')`? 
That will update all luge related elements.

**Reply by IsaMax** [reply]
Thank you  !

**Comment by IsaMax** [comment]
Hi ,
I have another question. I load content with ajax and the smooth scroll works fine. But the data-lg-parallax="child" on my image doesn't work after ajax call. How can i eventually refresh luge to have the parallax back ?
Thank you !

**Comment by AntoineW** [comment]
Hi,
That's a good point, right now you can't do that.

I updated the code to allow that. Could you test it with the files below?

You have to replace the `dist` folder in the `node_modules/@waaark/luge` folder by the one in the zip.
[luge-0.6.2-beta.zip](https://github.com/AntoineW/luge/files/7666935/luge-0.6.2-beta.zip)

Then you should be able to emit an `update` event like so `luge.emitter.emit('update')` after your ajax call.

Let me know if it works.

**Reply by IsaMax** [reply]
Hi, 
It works like a charm ! Thank you so much for your fix on that issue !

**Reply by AntoineW** [reply]
Thank you, I published the new version on npm 🙏
